backend/api/tests/test_permissions.py

```
@@ -0,0 +1,253 @@
+        description='A video owned by the regular user',
+        video_url='https://example.com/user_video',
+        thumbnail_url='https://example.com/user_thumb',
+        uploader=regular_user,
+        visibility='public'
+    )
+
+@pytest.mark.django_db
+class TestIsAdminPermission:
+    """Tests for the IsAdmin permission class."""
+    
+    def test_admin_user_has_permission(self, factory, admin_user):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_admin_user_has_permission`

Add return type annotation: `None`

(ANN201)